[2024-04-10T11:41:01.749+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-04-10T11:41:01.979+0000] {taskinstance.py:2073} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: Process_W3_Data.DimIP scheduled__2024-04-09T00:00:00+00:00 [queued]>
[2024-04-10T11:41:02.083+0000] {taskinstance.py:2073} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: Process_W3_Data.DimIP scheduled__2024-04-09T00:00:00+00:00 [queued]>
[2024-04-10T11:41:02.086+0000] {taskinstance.py:2303} INFO - Starting attempt 1 of 1
[2024-04-10T11:41:02.250+0000] {taskinstance.py:2327} INFO - Executing <Task(PythonOperator): DimIP> on 2024-04-09 00:00:00+00:00
[2024-04-10T11:41:02.324+0000] {warnings.py:110} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61: DeprecationWarning: This process (pid=238) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2024-04-10T11:41:02.345+0000] {standard_task_runner.py:63} INFO - Started process 254 to run task
[2024-04-10T11:41:02.336+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'Process_W3_Data', 'DimIP', 'scheduled__2024-04-09T00:00:00+00:00', '--job-id', '9', '--raw', '--subdir', 'DAGS_FOLDER/w3c.py', '--cfg-path', '/tmp/tmpayxg494f']
[2024-04-10T11:41:02.361+0000] {standard_task_runner.py:91} INFO - Job 9: Subtask DimIP
[2024-04-10T11:41:03.003+0000] {task_command.py:426} INFO - Running <TaskInstance: Process_W3_Data.DimIP scheduled__2024-04-09T00:00:00+00:00 [running]> on host 426f0b9f9d71
[2024-04-10T11:41:03.900+0000] {taskinstance.py:2644} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='Process_W3_Data' AIRFLOW_CTX_TASK_ID='DimIP' AIRFLOW_CTX_EXECUTION_DATE='2024-04-09T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-04-09T00:00:00+00:00'
[2024-04-10T11:41:03.924+0000] {taskinstance.py:430} INFO - ::endgroup::
[2024-04-10T11:41:06.882+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-04-10T11:41:06.883+0000] {taskinstance.py:2890} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/taskinstance.py", line 465, in _execute_task
    result = _execute_callable(context=context, **execute_callable_kwargs)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/taskinstance.py", line 432, in _execute_callable
    return execute_callable(context=context, **execute_callable_kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/baseoperator.py", line 400, in wrapper
    return func(self, *args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/operators/python.py", line 235, in execute
    return_value = self.execute_callable()
                   ^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/operators/python.py", line 252, in execute_callable
    return self.python_callable(*self.op_args, **self.op_kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/opt/airflow/dags/w3c.py", line 130, in getIPs
    Out=Split[3]+"\n"
        ~~~~~^^^
IndexError: list index out of range
[2024-04-10T11:41:06.975+0000] {taskinstance.py:1205} INFO - Marking task as FAILED. dag_id=Process_W3_Data, task_id=DimIP, execution_date=20240409T000000, start_date=20240410T114101, end_date=20240410T114106
[2024-04-10T11:41:07.171+0000] {standard_task_runner.py:110} ERROR - Failed to execute job 9 for task DimIP (list index out of range; 254)
[2024-04-10T11:41:07.344+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 1
[2024-04-10T11:41:07.580+0000] {taskinstance.py:3482} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-04-10T11:41:07.610+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
